lib/services/execution_iframe.dart

```diff
@@ -46,11 +50,27 @@ class ExecutionServiceIFrame implements ExecutionService {
     _send('setCss', {'css': css});
   }
 
+  /// TODO(redbrogdon): Format message so internal double quotes are escaped.
+  String get testResultDecoration => '''
+  void _result(bool success, String message) {
+    print('$testKey{"success": \$success, "message": "\$message"}');
```

Comment:
I don't think this works.  Consider:

  print('$testKey\{"success": $success, "message": ""$message"})');

Reply:
Fixed to show the escape in the right place.

Reply:
The string is getting interpolated twice (once to create a string constant when DartPad is run, and once when that constant is part of the Dart code that goes to dart-services to be compiled  into JS), which is why the escaping looks weird.